phpBB/phpbb/extension/extension_interface.php

```diff
@@ -27,6 +27,13 @@
 	public function is_enableable();
 
 	/**
+	* Indicate why the extension can not be enabled.
+	*
+	* @return string|array with actual processed language string(s) (not language keys), or empty
+	*/
+	public function not_enableable_reason();
```

Comment:
I think it's better to change the return of is_enableable() to void|string|array